Unable to switch context between projects

Hi,

We have started to use hcloud (v1.17.0) to manage 2 projects. However, we are not able to switch between them using context subcommand. It is possible to add and activate another project, however hcloud still makes requests to the first one.

After deleting the second project there is a segfault:
```
15:19 $ hcloud context list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7f7d46]

goroutine 1 [running]:
github.com/hetznercloud/cli/cli.runContextList(0xc0001f0180, 0xc0002478c0, 0xd57ab8, 0x0, 0x0, 0x0, 0x0)
	/Users/runner/runners/2.263.0/work/cli/cli/cli/context_list.go:60 +0x2c6
github.com/hetznercloud/cli/cli.(*CLI).wrap.func1(0xc0002478c0, 0xd57ab8, 0x0, 0x0, 0x0, 0x0)
	/Users/runner/runners/2.263.0/work/cli/cli/cli/cli.go:131 +0x56
github.com/spf13/cobra.(*Command).execute(0xc0002478c0, 0xc00018c020, 0x0, 0x0, 0xc0002478c0, 0xc00018c020)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:838 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001ae840, 0x0, 0x0, 0xc000280340)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:943 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:883
main.main()
	/Users/runner/runners/2.263.0/work/cli/cli/cmd/hcloud/main.go:35 +0x65
```

--- a/cli/context_list.go
+++ b/cli/context_list.go
@@ -57,7 +57,7 @@ func runContextList(cli *CLI, cmd *cobra.Command, args []string) error {
 			Token:  context.Token,
 			Active: " ",
 		}
-		if cli.Config.ActiveContext.Name == context.Name {
+		if cli.Config.ActiveContext != nil && cli.Config.ActiveContext.Name == context.Name {
 			presentation.Active = "*"
 		}
 

